docs/src/docs/asciidoc/working-with-asciidoctor.adoc

```diff
@@ -150,55 +164,63 @@ snippet is included or by using a custom snippet template.
 
 
 [[working-with-asciidoctor-customizing-tables-formatting-columns]]
-==== Formatting columns
+==== Formatting Columns
 
 Asciidoctor has rich support for
-http://asciidoctor.org/docs/user-manual/#cols-format[formatting a table's columns]. For
-example, the widths of a table's columns can be specified using the `cols` attribute:
+http://asciidoctor.org/docs/user-manual/#cols-format[formatting a table's columns]. As the
+following example shows, you can specify the widths of a table's columns by using the
+`cols` attribute:
 
+====
 [source,indent=0]
 ----
 [cols="1,3"] <1>
 \include::{snippets}/index/links.adoc[]
 ----
-<1> The table's width will be split across its two columns with the second column being
+<1> The table's width is split across its two columns, with the second column being
 three times as wide as the first.
+====
 
 
 
 [[working-with-asciidoctor-customizing-tables-title]]
-==== Configuring the title
+==== Configuring the Title
 
-The title of a table can be specified using a line prefixed by a `.`:
+You can specify the title of a table by using a line prefixed by a `.`.
+The following example shows how to do so:
 
+====
 [source,indent=0]
 ----
 .Links <1>
 \include::{snippets}/index/links.adoc[]
 ----
 <1> The table's title will be `Links`.
+====
 
 
 
 [[working-with-asciidoctor-customizing-tables-formatting-problems]]
-==== Avoiding table formatting problems
+==== Avoiding Table Formatting Problems
 
 Asciidoctor uses the `|` character to delimit cells in a table. This can cause problems
-if you want a `|` to appear in a cell's contents. The problem can be avoided by
-escaping the `|` with a backslash, i.e. by using `\|` rather than `|`.
+if you want a `|` to appear in a cell's contents. You can avoide the problem by
```

Comment:
avoid

Reply:
Wow. I was on a typo roll. Thanks for catching those.